reana-ui/src/components/WorkflowProgressCircleBar.js

```diff
@@ -0,0 +1,103 @@
+/*
+  -*- coding: utf-8 -*-
+
+  This file is part of REANA.
+  Copyright (C) 2024 CERN.
+
+  REANA is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+import PropTypes from "prop-types";
+import styles from "./WorkflowProgressCircleBar.module.scss";
+
+export default function WorkflowProgressCircleBar({ workflow }) {
+  const { completed, failed, running, total, status } = workflow;
+
+  const size = 80;
+  const strokeWidth = 10;
+  const radius = size / 2 - strokeWidth;
```

Comment:
```suggestion
  const radius = size / 2 - strokeWidth / 2;
```
strokeWidth probably also needs to be divided by 2, as the radius is calculated fro the center to the "middle" of the stroke (in other words the inner circle has radius r - strokeWidth/2, and the outer radius is r + strokeWidth/2). Otherwise the circle will be smaller than the size of the svg:
![Schermata 2024-02-28 alle 12 41 13](https://github.com/reanahub/reana-ui/assets/13397702/ede67406-287f-4b00-a78d-cb1c80806de3)